### DISCUSSION TITLE
Where is the downloaded ch_model.safetensors stored in Ubuntu?

### DISCUSSION THREAD

**Discussion by marks202309** [bmaltais/kohya_ss#1581] [Q&A]
Downloading (…)ain/model_index.json: 100%|███████████████████████████████████████████████████████████████████████████████████| 541/541 [00:00<00:00, 3.28MB/s]
Downloading (…)rocessor_config.json: 100%|███████████████████████████████████████████████████████████████████████████████████| 342/342 [00:00<00:00, 2.14MB/s]
Downloading (…)5b9/unet/config.json: 100%|███████████████████████████████████████████████████████████████████████████████████| 743/743 [00:00<00:00, 11.3MB/s]
Downloading (…)cheduler_config.json: 100%|███████████████████████████████████████████████████████████████████████████████████| 308/308 [00:00<00:00, 4.70MB/s]
Downloading (…)b5b9/vae/config.json: 100%|███████████████████████████████████████████████████████████████████████████████████| 547/547 [00:00<00:00, 3.38MB/s]
Downloading (…)_encoder/config.json: 100%|███████████████████████████████████████████████████████████████████████████████████| 617/617 [00:00<00:00, 2.58MB/s]
Downloading (…)ch_model.safetensors: 100%|█████████████████████████████████████████████████████████████████████████████████| 335M/335M [02:21<00:00, 2.37MB/s]
Downloading model.safetensors: 100%|███████████████████████████████████████████████████████████████████████████████████████| 492M/492M [03:13<00:00, 2.55MB/s]
Downloading (…)ch_model.safetensors: 100%|███████████████████████████████████████████████████████████████████████████████| 3.44G/3.44G [06:31<00:00, 8.78MB/s]
Fetching 9 files: 100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████| 9/9 [06:33<00:00, 43.73s/it]
Loading pipeline components...: 100%|███████████████████████████████████████████████████████████████████████████████████████████| 5/5 [00:00<00:00, 15.29it/s]

**Comment by TashinAhmed** [comment]
@marks202309 

It's under the cache directory, suppose to be under your `user_profile/.cache`
